ansible/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -65,7 +65,7 @@ listen_addresses = '*'      # what IP address(es) to listen on;
                     # defaults to 'localhost', '*' = all
                     # (change requires restart)
 port = 5432             # (change requires restart)
-max_connections = 100           # (change requires restart)
+max_connections = 300           # (change requires restart)
```

Comment:
I don't think this number is to high though don't have anything to back
that up with. I was going to look at this next week and see if I could find
anything in the Apache logs leading up to the would indicate cause.

Ultimately though I think we should install pgbouncer which handles
connection pooling etc.
On 15 May 2015 18:59, "Ben" <notifications@github.com> wrote:

> In ansible/roles/postgresql/templates/postgresql.conf.j2
> <https://github.com/dimagi/commcarehq-ansible/pull/93#discussion_r30425895>
> :
>
> > @@ -65,7 +65,7 @@ listen_addresses = '*'      # what IP address(es) to listen on;
> >                      # defaults to 'localhost', '*' = all
> >                      # (change requires restart)
> >  port = 5432             # (change requires restart)
> > -max_connections = 100           # (change requires restart)
> > +max_connections = 300           # (change requires restart)
>
> @snopoke <https://github.com/snopoke> do you think this number is
> actually too high? everytime postgres goes tits up we have to kill a bunch
> of connections. trying to do a bit of research into it, but it seems like a
> complicated question that depends on a variety of factors.
> http://serverfault.com/a/482302/246939
> https://wiki.postgresql.org/wiki/Tuning_Your_PostgreSQL_Server
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcarehq-ansible/pull/93/files#r30425895>.
>